src/logic/exceptions/registry.ts

```diff
@@ -9,6 +9,7 @@ enum ErrorCodes {
   _100 = '100: Invalid input in the address field',
   _600 = '600: Error fetching token list',
   _601 = '601: Error fetching balances',
+  _602 = '602: Error processing Safe Apps request',
```

Comment:
Could you start a new hundred for Safe Apps errors? Like 9xx.
We want to try and group them as per the comment in the comment above.